app/controllers/logout_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class LogoutController < ApplicationController
```

Comment:
```suggestion
class Users::LogoutController < ApplicationController
```

Could we put this in the users module?